src/Scrutor/IImplementationTypeFilter.cs

```diff
@@ -135,5 +135,12 @@ public interface IImplementationTypeFilter : IFluentInterface
         /// <param name="predicate">The predicate to match types.</param>
         /// <exception cref="ArgumentNullException">If the <paramref name="predicate" /> argument is <c>null</c>.</exception>
         IImplementationTypeFilter Where(Func<Type, bool> predicate);
+
+        /// <summary>
+        /// Filter out types where the namespace is not or a child of the provided one.

```

Comment:
I've had cases where the Interface and the implementation I wanted was in an "impl" sub namespace as well as something else that implemented the interface way off somewhere else (like a little test proj).  This has screwed up the registration.  If you feel it's redundant, I can remove this.  I wasn't in love with it.